gson/src/main/java/com/google/gson/stream/JsonWriter.java

```diff
@@ -225,6 +230,22 @@ public final void setIndent(String indent) {
     }
   }
 
+  /**
+   * Sets the newline string to be used in the encoded document.
```

Comment:
Would be good to clarify that this has no effect if no indent (with link to `#setIndent`) is set.

Reply:
Done.

I used the more vague description _"Has no effect if the serialized format is a single line."_
because I am not sure on all the conditions.
For example `gson.toJson("test")` does not need a newline, no matter if we are using indent or not.

And if a condition changes, then the doc will be out of sync.
In the future the example above this might change to "use trailing newline if pretty-print", who knows :-)